### PR TITLE
Pr/dms upload

### PR DESCRIPTION
**Pull request by stewue**

### CODE REVIEW

File: packages/tocco-ui/src/Upload/UploadInput.js [hunk 1]
```diff
@@ -25,6 +25,20 @@ const UploadInput = ({onDrop: onDropProp, immutable, text}) => {
               position={design.position.PREPEND}
             />
           </Button>
+          {onChoose && <Button
+
```
Comment (rzueger) [inline packages/tocco-ui/src/Upload/UploadInput.js:29]: (blank line can be removed)

File: packages/tocco-ui/src/Upload/UploadInput.js [hunk 2]
```diff
@@ -25,6 +25,20 @@ const UploadInput = ({onDrop: onDropProp, immutable, text}) => {
...
+              e => {
+                onChoose()
+                e.stopPropagation()
+              }
+            }
+          >
+            <Icon
+              icon="folder"
+              position={design.position.PREPEND}
+            />
```
Comment (rzueger) [inline packages/tocco-ui/src/Upload/UploadInput.js:40]: Now that there are two icon buttons, i think they both should have a title that is shown on mouseover (e.g. "choose from computer" and "choose from uploaded documents" or something like that)

File: packages/app-extensions/src/formData/FormDataContainer.js [hunk 3]
```diff
@@ -59,6 +59,10 @@ const mapStateToProps = (
     ? {navigationStrategy: state.formData.navigationStrategy.navigationStrategy}
     : {}
   ),
+  ...(state.formData.upload && state.formData.upload.chooseDocument
```
Comment (rzueger) [inline packages/app-extensions/src/formData/FormDataContainer.js:62]: could this condition be shortened to `state.formData.upload?.chooseDocument`?

File: packages/docs-browser/src/modules/chooseDocument/sagas.spec.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import {expectSaga, testSaga} from 'redux-saga-test-plan'
+import {takeLatest} from 'redux-saga/effects'
+import {notification} from 'tocco-app-extensions'
+
+import * as sagas from './sagas'
+import * as actions from './actions'
+
+describe('admin', () => {
+  describe('routes', () => {
+    describe('docs', () => {
```
Comment (rzueger) [inline packages/docs-browser/src/modules/chooseDocument/sagas.spec.js:10]: (wrong namespace admin/routes/docs?)

**Review by rzueger — APPROVED** [review]

File: packages/tocco-ui/src/Upload/UploadInput.js (hunk 2 above)
Comment (stewue) [reply packages/tocco-ui/src/Upload/UploadInput.js:39]: just setting a title on the icon does not work. there is a new task for yilmaz to fix several styling things

Comment (rzueger) [reply packages/tocco-ui/src/Upload/UploadInput.js:39]: what about the `title` attribute of the `Button`? doesn't work either?

Comment (stewue) [reply packages/tocco-ui/src/Upload/UploadInput.js:39]: also doesn't work

**Pull request merged**